content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -2,94 +2,98 @@
-are two ways to configure the appropriate BoM for the respective product combination. 
+As suggested above, you can assign BoMs to specific *Product Variants*. There are two ways to set
+up BoMs for product variants, after the attributes have been configured on the product form.
 
-Either create one BoM per variant, by specifying the Product Variant
-in the dedicated field below the product name. Or use one BOM, that contains
-all of the components and for each component, indicate which variant it applies to, 
-using the *Apply on Variants* column, as shown below. 
+You can create one BoM per variant by specifying the *Product Variant* in the dedicated field below
+the product name. Alternatively, you can use one master BoM that contains all of the components, and
+specify which variant each component applies to using the *Apply on Variants* column, as shown
+below. 
 
 .. image:: bill_configuration/bom_2.png
     :align: center
+    :alt: Product Variants in the Bill of Materials. 
 
 
 Adding Operations
 =================
 
-You can also add operations to your *BoM*, if you want workers to follow instructions
-or register time spent. To use this feature, enable the *Work Orders* feature in the 
-*Manufacturing* app settings, as shown below.
+You can also add *Operations* to your BoM if you want workers to follow instructions or register
+time spent. To use this feature, enable the *Work Orders* feature in the *Manufacturing* app
+settings, as shown below.
 
 .. image:: bill_configuration/bom_3.png
     :align: center
+    :alt: Work Orders Setting. 
 
 .. note::
-         Each operation is unique as it is always linked to only one BOM. This being said, 
-         Operations Operations can be re-used when configuring a new BOM, with the *Copy Existing 
+         Each operation is unique, as it is always exclusively linked to one BoM. 
+         Operations can be re-used when configuring a new BoM, with the *Copy Existing 
          Operations* feature.
 
 .. image:: bill_configuration/bom_4.png
     :align: center
+    :alt: *Copy Existing Operations* feature. 
 
-Finally, just like for components, operations can also be variant specific only, as shown below.
+Like components, operations can also be variant-specific, as shown below.
 
 .. image:: bill_configuration/bom_5.png
     :align: center
+    :alt: Variant-specific operations. 
 
 
 
 Adding By-Products
 ==================
 
-A *by-product* is a product that is produced on top of the main product 
-of a *BoM*. As opposed to the primary product, there can be more than one on a BOM. 
+A *By-Product* is a product that is produced in addition to the main product of a BoM. Unlike the
+primary product, there can be more than one by-product on a BoM. 
 
-To add *by-products* to a *BoM*, you will first need to enable the by-product
-feature from the *Manufacturing* app settings.
+To add by-products to a BoM, you will first need to enable the *By-Products* feature from the
```

Comment:
- Delete `you will` and `need to`
- Replace the italic formatting around `*By-Products*` with `:guilabel:'By-Products'`
- Replace `from the *Manufacturing* app settings` with `in :menuselection:'Manufacturing --> Configuration --> Settings --> Operations'`

Note: in my suggested changes, single quotation marks ( ' ) = backticks ( ` ), I use single quotation marks in review comments so it will actually show on Github